Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/CertificatesAdapter.kt

```diff
@@ -37,6 +39,8 @@ class CertificatesAdapter :
                 TypedVHCreatorMod({ data[it] is CreateVaccinationCard.Item }) { CreateVaccinationCard(it) },
                 TypedVHCreatorMod({ data[it] is HeaderInfoVaccinationCard.Item }) { HeaderInfoVaccinationCard(it) },
                 TypedVHCreatorMod({ data[it] is BottomInfoVaccinationCard.Item }) { BottomInfoVaccinationCard(it) },
+                TypedVHCreatorMod({ data[it] is VaccinationTestSuccessCard.Item }) { VaccinationTestSuccessCard(it) },
+                TypedVHCreatorMod({ data[it] is VaccinationTestErrorCard.Item }) { VaccinationTestErrorCard(it) },
```

Comment:
Maybe we should rename the cards here , those cards are not for Vaccination right? 
`CovidTestCertificateCard`,  `CovidTestCertificateErrorCard(it)` ? or whatever you think it is better ?

Reply:
renamed